README.md

```diff
@@ -464,50 +464,32 @@ By default, Snowflake returns an error for queries issued with multiple statemen
 
 ---
 
-You can execute multiple statements as a batch in the same way you execute queries with single statements, except that the query string contains multiple statements separated by semicolons. Note that multiple statements execute sequentially, not in parallel. The MULTI_STATEMENT_COUNT parameter lets you specify the exact number of statements the batch contains.
+You can execute multiple statements as a batch in the same way you execute queries with single statements, except that the query string contains multiple statements separated by semicolons. Note that multiple statements execute sequentially, not in parallel. The MULTI_STATEMENT_COUNT parameter lets you enable or disable support for executing a batch of statements:
 
-For example, if you set MULTI_STATEMENT_COUNT=3, a batch statement must include precisely three statements. If you submit a batch statement with any other number of statements, the Node.js driver rejects the request. You can set MULTI_STATEMENT_COUNT=0 to allow batch queries to contain any number of statements. However, be aware that using this value reduces the protection against SQL injection attacks.
-
-You can set this parameter at the session level using the following command, or you can set the value separately each time you submit a query.
+You can set this parameter at the session level using the following command:
 
 ```
 ALTER SESSION SET MULTI_STATEMENT_COUNT = <n>;
 ```
 
-By setting the value the session level, you do not need to set it when you execute each time you execute a batch statement. The following example sets the number of statements at the session level to three and then executes three SQL statements:
```

Comment:
Sorry for the misleading. I think we need some correction here. `By setting the value the session level, you do not need to set it when you execute each time you execute a batch statement.` This is true and need to be mentioned in some way, since setting MULTI_STATEMENT_COUNT=0 in session level is not just enable multiple statements feature, it allows any number of statements in a query, and that could be vulnerable to SQL injection. Better to mention that and suggest setting number of statements per query through is a more safe way.

Reply:
@sfc-gh-mheavin In another word, using parameter to set number of statements per query doesn't require setting MULTI_STATEMENT_COUNT=0 in session level, it works when MULTI_STATEMENT_COUNT=1 in session level so basically the we suggest customer to keep MULTI_STATEMENT_COUNT=1 in session level and set number of statements per query through parameter.

Reply:
Rewrote section for review.